docs/UserGuide.md

```diff
@@ -306,6 +322,11 @@ To sort by number of properties in the default order, but with buyers shifted to
 
 Opens a new window and shows all sellers and buyers with matching property and preference.
 
+A preference matches with a property if 
+- they have the same `region`, and
+- they have the same `size`, and
+- the `price` of the property is between `lowPrice` and `highPrice` of the preference. 
+
```

Comment:
perhaps you can clarify if the `lowPrice` & `highPrice` of the preference is **inclusive**? 

meaning is the check using `propertyPrice >= lowPrice && propertyPrice <= highPrice` or `propertyPrice > lowPrice && propertyPrice < highPrice`

Reply:
Changes made!